ports/stm32/Makefile

```diff
@@ -258,6 +258,8 @@ SRC_C = \
 	servo.c \
 	dac.c \
 	adc.c \
+	ethernetif.c \
+	network_ethernet.c \
```

Comment:
Where did you obtain this file from?

Reply:
Specifically, the network_ethernet.c file

Reply:
As mentioned above, this PR was superseded by #4541 and `network_ethernet.c` was replaced by `network_lan.c`